samples/client/petstore/typescript-node/default/.swagger-codegen/VERSION

```diff
@@ -1 +1 @@
-2.3.1
\ No newline at end of file
+unset
```

Comment:
should be 2.4.0-SNAPSHOT

Reply:
This was quite strange for me. Whenever I have regenerated clients/servers in the past this version has been automatically set when running the generation scripts in /bin. I'm wondering whether something has changed/broken. Any ideas?